OSSSpeechKit/Classes/OSSVoice.swift

```diff
@@ -226,12 +229,14 @@ public enum OSSVoiceEnum: String, CaseIterable {
     /// You can supply your own flag image, provided is has the same name (.rawValue) as the image in the pod assets.
     ///
     /// If no image is found in the application bundle, the image from the SDK bundle will be provided.
+	#if canImport(UIKit)
```

Comment:
Indentation is incorrect on the `#if` here as well.

Reply:
Fixing. I've seen it both ways though. Swift-format allows indenting though Xcode by default doens't indent them.